services/community-site/src/pages/Staking/Modal/Delegate.tsx

```diff
@@ -117,20 +118,21 @@ const Delegate = ({ balance, validator, ownDelegation, onSuccess, onFinish }: De
           fullWidth
           margin="normal"
           onChange={(event) => {
+            setDelegationTotal(event.target.value);
+          }}
+          onKeyUp={(event) => {
+            const inputValue = (event.target as HTMLInputElement).value;
             if (parseFloat(delegationTotal) > parseFloat(ethers.utils.formatEther(balance))) {
```

Comment:
Shouldn't we also check the inputValue here and not the delegationTotal?

Reply:
Yeah, missed it.